src/Game.js

```diff
@@ -0,0 +1,80 @@
+class Game {
+	constructor(frame = new Frame()) {
+		this.frameNo = 1;
+		this.frames = [];
+		this.currentFrame = frame;
+		this.currentscore = 0;
+		this.gameOver = false;
+	}
+
+	nextFrame(frame = new Frame()) {
+		if (this.frameNo > 10) {
+			return (this.gameOver = true);
+		} else {
+			this.frames.push(this.currentFrame);
+			this.frameNo++;
+			if (this.frameNo < 10) {
+				this.currentFrame = frame;
```

Comment:
`this.currentFrame` could be a method that returns`this.frames[this.frames.length - 1]`